screenpy/actor.py

```diff
@@ -78,38 +86,62 @@ def has_ordered_cleanup_tasks(self, *tasks: Performable) -> "Actor":
         The tasks given to this method must be performed successfully in
         order. If any task fails, any subsequent tasks will not be attempted
         and will be discarded.
+
+        Aliases:
+            * :meth:`~screenpy.actor.Actor.has_cleanup_task`
+            * :meth:`~screenpy.actor.Actor.with_cleanup_task`
+            * :meth:`~screenpy.actor.Actor.with_ordered_cleanup_tasks`
         """
         self.ordered_cleanup_tasks.extend(tasks)
         return self
 
-    has_cleanup_task = has_ordered_cleanup_tasks
-    with_cleanup_task = with_ordered_cleanup_tasks = has_ordered_cleanup_tasks
+    def has_cleanup_task(self, *tasks: Performable) -> "Actor":
+        """Alias for :meth:`~screenpy.actor.Actor.has_ordered_cleanup_tasks`."""
+        return self.has_ordered_cleanup_tasks(*tasks)
+
+    def with_cleanup_task(self, *tasks: Performable) -> "Actor":
+        """Alias for :meth:`~screenpy.actor.Actor.has_ordered_cleanup_tasks`."""
+        return self.has_ordered_cleanup_tasks(*tasks)
+
+    def with_ordered_cleanup_tasks(self, *tasks: Performable) -> "Actor":
+        """Alias for :meth:`~screenpy.actor.Actor.has_ordered_cleanup_tasks`."""
+        return self.has_ordered_cleanup_tasks(*tasks)
```

Comment:
I propose we simply drop the singular variants of these methods because it's purely an English detail. 
That would render a total of 6 aliases comprising both types of cleanup tasks.

(or even better, 4 total if we just drop the `ordered` aliases and rely on `independent` cleanup tasks to already be distinguishing themselves from the implied ordered ones... but that's a separate discussion.)

If we do want to explicitly alias singular and plural forms as you have here then we should also include all the options by adding `has_ordered_cleanup_task`, `has_independent_cleanup_task`, and `with_independent_cleanup_task`.

Personally I'd rather just accept a format in which `has_{type}_cleanup_tasks()` may contain one or more tasks, but you know my position is that English readability doesn't have to mean _native speaker_ english readability in terms of precise tense agreement

```python
the_actor.has_ordered_cleanup_tasks(ClickButton())
the_actor.has_independent_cleanup_tasks(
    ClearTextField(),
    LogOut(),
)
```

Reply:
TL:DR - 

A) (my preference) Dropping the singulars would leave us with:
```python
has_cleanup_tasks()
has_ordered_cleanup_tasks()
with_cleanup_tasks()
with_ordered_cleanup_tasks()

has_independent_cleanup_tasks()
with_independent_cleanup_tasks()
```

B) (something to consider) Also dropping the `ordered` variants and relying on the existing distinction already evident by the `independent` methods would leave us with
```python
has_cleanup_tasks()
with_cleanup_tasks()

has_independent_cleanup_tasks()
with_independent_cleanup_tasks()
```

C) (english stickler) And going the other direction with all singular and plural options would leave us with
```python
has_cleanup_task()
has_cleanup_tasks()
with_cleanup_task()
with_cleanup_tasks()
has_ordered_cleanup_task()
has_ordered_cleanup_tasks()
with_ordered_cleanup_task()
with_ordered_cleanup_tasks()

has_independent_cleanup_task()
has_independent_cleanup_tasks()
with_independent_cleanup_task()
with_independent_cleanup_tasks()
```

Reply:
I suppose we could go a different way with option (B) by dropping the less explicit form of the ordered variants and just stick with only the plurals of each e.g.
```python
has_ordered_cleanup_tasks()
with_ordered_cleanup_tasks()

has_independent_cleanup_tasks()
with_independent_cleanup_tasks()
```

Reply:
Haha, this is a great point. I think i agree with the streamline.